symengine/llvm_double.h

```diff
@@ -52,13 +52,14 @@ class LLVMVisitor : public BaseVisitor<LLVMVisitor>
 public:
     llvm::Value *apply(const Basic &b);
     void init(const vec_basic &x, const Basic &b,
-              const bool symbolic_cse = false, int opt_level = 2);
+              const bool symbolic_cse = false, unsigned opt_level = 3);
     void init(const vec_basic &x, const Basic &b, const bool symbolic_cse,
-              const std::vector<llvm::Pass *> &passes);
+              const std::vector<llvm::Pass *> &passes, unsigned opt_level = 3);
```

Comment:
@ichumuh, could you add a test for this new interface as well? Sorry, I'm busy now that the semester has started.